NetworkRegistry/schema.json

```diff
@@ -0,0 +1,31 @@
+{
+    "definitions": {
+      "network": {
+        "$id": "#/definitions/network",
+        "type": "object",
+        "properties": {
+          "Name": {
+            "type": "string"
+          },
+          "NetworkId": {
+            "type": "integer"
+          },
+          "NetworkMagic": {
+            "type": "integer"
+          },
+          "GenesisHash": {
+            "type": "string"
```

Comment:
Possibly, we could add `"encoding": "base16", "minLength":  64, "maxLength": 64`